ibis/expr/datatypes.py

```diff
@@ -605,6 +606,140 @@ def _equal_part(self, other, cache=None):
         )
 
 
+# GEO Spatial Data Types
+
+class GeoSpatial(DataType):
+
+    __slots__ = ()
+
+    def __repr__(self):
+        name = self.name.lower()
+        if not self.nullable:
+            return '{}[non-nullable]'.format(name)
+        return name
+
+
+class Point(GeoSpatial):
+    scalar = ir.PointScalar
+    column = ir.PointColumn
+
+    __slots__ = 'x', 'y'
+
+    def __init__(self, x, y, nullable=True):
```

Comment:
I think you're misunderstanding how `DataType`s work. `DataType`s are just type information, and shouldn't be used to construct values. The way a user would make a point object might be with a 2-tuple:

```python
ibis.literal((1, 2), type='point')
```

Reply:
oh I see .. thanks for the explanation!